messenger.rst

```diff
@@ -1629,7 +1629,7 @@ The Redis transport DSN may looks like this:
     # Unix Socket Example
     MESSENGER_TRANSPORT_DSN=redis:///var/run/redis.sock
     # TLS Example
-    MESSENGER_TRANSPORT_DSN=rediss://localhost:6379/messages
+    MESSENGER_TRANSPORT_DSN=redis://localhost:6379/messages
```

Comment:
`rediss` is a valid scheme to enable TLS

Reply:
Ah TIL!